CONTRIBUTING.md

```diff
@@ -259,3 +260,9 @@ To trigger production rebuild and redeploy, use personal GitHub token and run:
 ```shell
 GITHUB_TOKEN=<your token> npm run make-empty-commit -- --message "trigger deploy" --branch main
 ```
+
+## Get Support
+
+We have a thriving community on discord. If you have any questions related to
+contributing please reach out, we'd be happy to help!
+[Join us on discord to start the conversation](https://discord.gg/cypress).
```

Comment:
```suggestion
We have a thriving community on Discord. If you have any questions related to
contributing please reach out, we'd be happy to help!
[Join us on Discord to start the conversation](https://discord.gg/cypress).
```
Discord is written uppercase.

There are different links in use for Cypress on Discord:

https://on.cypress.io/discord

is also used in https://github.com/cypress-io/cypress/blob/develop/.github/ISSUE_TEMPLATE/1-bug-report.yml

The support page lists

https://discord.com/invite/cypress

This should be checked, which is the preferred Discord link to be passed on.

Reply:
The discord channel is not monitored for helping individuals contribute to Cypress. It would be more appropriate to remove add the Discord chat badge to the README. See: https://github.com/cypress-io/cypress#readme

Also the best link to use is `on.cypress.io/chat`. This is managed by our team to direct users to the correct invite if/when it expires.

Reply:
@emilyrohrbough 

I was a bit confused about "It would be more appropriate to remove add the Discord chat badge to the README. See: https://github.com/cypress-io/cypress#readme". Did you mean "remove" or "add"?

The README contains

  <a href="https://on.cypress.io/discord">
    <img src="https://img.shields.io/badge/chat-on%20Discord-brightgreen" alt="Discord chat"/>
  </a>

which links to https://on.cypress.io/discord

Are you saying that the link in the README should be changed to 
 https://on.cypress.io/chat
instead?

Reply:
@emilyrohrbough 

> I am suggesting removing the suggested changes to the contribution guide all together and adding the badge to the readme instead.

I agree that adding this information to CONTRIBUTING.md is the wrong place. The discussions which take place in Discord are about using Cypress. I have not seen discussions there about contributing to the development of Cypress in terms of PRs to the software or to the documentation.

The [README](https://github.com/cypress-io/cypress-documentation/blob/main/README.md) in this (documentation) repo could copy the badge used in the [README](https://github.com/cypress-io/cypress/blob/main/README.md) of the main repo.

Additionally (and probably outside of the scope of this PR), there could be a section "How to get support" added to
- the docu README
- the main Cypress README
- the footer of the docu page https://docs.cypress.io/

each linking to the detailed webpage  https://www.cypress.io/support/ how to get support.